packages/lib/services/interop/InteropService_Exporter_Md.ts

```diff
@@ -4,7 +4,9 @@ import shim from '../../shim';
 import markdownUtils from '../../markdownUtils';
 import Folder from '../../models/Folder';
 import Note from '../../models/Note';
+import { NoteEntity } from '../database/types';
 import { basename, dirname, friendlySafeFilename } from '../../path-utils';
+const { MarkupToHtml } = require('@joplin/renderer');
```

Comment:
Since it's a TypeScript file, I think you can use `import { MarkupToHtml } from '@joplin/renderer'` to get typing.